icu4j/pom.xml

```diff
@@ -93,6 +93,18 @@
         <plugin>
           <artifactId>maven-surefire-plugin</artifactId>
           <version>3.1.2</version>
+          <configuration>
+            <includes>
+              <include>**/Test*.java</include>
+              <include>**/*Test.java</include>
+              <include>**/*Tests.java</include>
+              <include>**/*TestCase.java</include>
+              <!-- Some test files start with "IntlTest...". Many don't fit
+               the Surefire test file name default patterns. To be safe, use
+               a catchall pattern. -->
+              <include>**/*.java</include>
```

Comment:
Do we need the first four includes when we have the last one as a "catchall"?

Reply:
No, they're redundant. I copied them because those are the plugin's defaults for the includes. Should I remove them?

Reply:
I think that the unexplained redundancy is confusing. I suggest you do one of
- remove the subset includes
- comment out the subset includes

Either way, please add a comment saying why we don't just use the default (maybe with a link to the default, or with an example).

Reply:
Done.